python/llm/dev/benchmark/ceval/eval.py

```diff
@@ -0,0 +1,308 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import argparse
+import pandas as pd
+import torch
+import json
+from tqdm import tqdm
+
+import intel_extension_for_pytorch as ipex
```

Comment:
why import ipex here? We should also support cpu?
Import when xpu in device?

Reply:
This import can be removed, because current we will import ipex automatically.

Reply:
cool